bubblesub/util.py

```diff
@@ -16,8 +16,10 @@
 
 """Miscellaneous functions and classes for general purpose usage."""
 
+import ast
 import fractions
 import itertools
+import operator
```

Comment:
Again this was about deferring complex imports so that splash screen can load faster

Reply:
No significant difference on my machine. Is it a problem on yours?

Reply:
Nah